modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -942,20 +942,88 @@ public String getAlias(String name) {
         return name;
     }
 
+    /**
+     * Return the default value of array property
+     * <p>
+     * Return null if there's no default value.
+     * Any non-null value will cause {{#defaultValue} check to pass.
+     *
+     * @param cp Codegen property
+     * @param schema Property schema
+     * @return string presentation of the default value of the property
+     */
+    public String toArrayDefaultValue(CodegenProperty cp, Schema schema) {
+        if (schema.getDefault() != null) { // has default value
+            if (cp.isArray && !cp.getUniqueItems()) { // array
+                List<String> _values = new ArrayList<>();
+
+                if (schema.getDefault() instanceof ArrayNode) { // array of default values
+                    ArrayNode _default = (ArrayNode) schema.getDefault();
+                    if (_default.isEmpty()) {
+                        return "null";
+                    }
+
+                    List<String> final_values = _values;
+                    _default.elements().forEachRemaining((element) -> {
+                        final_values.add(element.asText());
+                    });
+                } else { // single value
+                    _values = Arrays.asList(String.valueOf(schema.getDefault()));
+                }
+
+                String defaultValue = "";
+
+                if (cp.items.isEnum) { // enum
+                    List<String> defaultValues = new ArrayList<>();
+                    for (String _value : _values) {
+                        defaultValues.add(cp.items.datatypeWithEnum + "." + toEnumVarName(_value, cp.items.dataType));
+                    }
+                    defaultValue = StringUtils.join(defaultValues, ", ");
+                } else {
+                    if (cp.items.isString) {
+                        defaultValue = StringUtils.join(_values, "\", \"");
+                        defaultValue = "\"" + defaultValue + "\"";
```

Comment:
```suggestion
                        defaultValue = _values.stream().collect(Collectors.joining("\", \"", "\"", "\""));
```

Reply:
FYI. Reverted this change as the output is not the same anymore.